### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by tothandras**
Sorry for the huge PR...

### CODE REVIEW

File: eslint/eslint-defaults.yaml [hunk 1]
```diff
@@ -244,7 +235,7 @@ rules:
   space-infix-ops:
     - 2
     - int32Hint: false
-  space-return-throw-case: 2
+  keyword-spacing: 2
   space-unary-ops:
     - 2
     - words: true
```
Comment (ksiegel) [inline eslint/eslint-defaults.yaml]: :+1: on all these changes!

**Comment by ksiegel** [comment]
Can we cherry-pick the eslint changes to master to make this PR diff easier to review?

**Comment by tothandras** [comment]
The tests are passing now, sadly eslint and formatting changes can be found in most of the commits.

**Comment by tothandras** [comment]
Please review :) We would like to merge this, because it starts to get out of hands.

**Comment by tothandras** [comment]
We'll do standalone PRs for every fix, feature from now on.

**Pull request merged**